src/templates/test.html

```diff
@@ -1,27 +1,36 @@
 <!-- Bootstrap HTML for running the unit tests. -->
 <html>
-    <head>
-        <meta charset="utf-8"/>
-        <script>
-         window.logs = [];
-         console.log = function(message) {
-             window.logs.push(message);
-         }
-         console.warn = function(message) {
-             window.logs.push(message);
-         }
-         console.info = function(message) {
-             window.logs.push(message);
-         }
-         console.error = function(message) {
-             window.logs.push(message);
-         }
-        </script>
-        <script type="text/javascript">
-            // set the pyodide files URL (packages.json, pyodide.asm.data etc)
-            window.languagePluginUrl = './';
-        </script>
-        <script src="./pyodide.js"></script>
-    </head>
-    <body></body>
+  <head>
+  <meta charset="utf-8"/>
+  <script type="text/javascript">
+    window.logs = [];
+    console.log = function(message) {
```

Comment:
If python is printing to stdout/err instead of calling `console.log` via `js` directly, I think the correct thing to do is to override `Module.print` and `Module.printErr`

Reply:
Maybe we should do both?

Reply:
I would prefer not to override console.log, in general, as that could have surprising consequences down the road.

Reply:
Just to be clear, though, that was what was in there *before* this change.

Reply:
Oh right you simply re-formatted it. We can change it in a separate PR then